api/app/services/system_upgrades/manifold000400.rb

```diff
@@ -0,0 +1,41 @@
+module SystemUpgrades
+  class Manifold000400 < ActiveInteraction::Base
+    object :logger
+
+    def execute
+      logger.info "Updating Project tweet_fetch_config"
+      reformat_tweet_fetch_config
+    end
+
+    def reformat_tweet_fetch_config
```

Comment:
Probably all methods other than `execute` should be private.

Reply:
Damnit, I was going to do that.  Updating.

Reply:
:)